t/t3404-rebase-interactive.sh

```diff
@@ -397,7 +400,9 @@ test_expect_success 'multi-squash only fires up editor once' '
 			git rebase -i $base
 	) &&
```

Comment:
Hi @sunshineco thanks for the review. I really appreciate. I have a couples of doubt to clear. 

1. My next patch should the squash of my three patches which include my first two patch and the new one on the same branch ?
2. Two patch series means two different commit on different patch ? but, since they both depends on each other would not they leads to merge  conflict ?

Thank you.

Reply:
Please ask these questions in a response to my email on the Git mailing list so that other readers can easily follow and engage in the discussion.

Reply:
@sunshineco, thanks,  I am now getting more familiar with the Git mailing list.